server/integration-tests/builders/src/test/java/io/quarkiverse/openapi/server/generator/it/PetStoreTest.java

```diff
@@ -0,0 +1,44 @@
+package io.quarkiverse.openapi.server.generator.it;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.anything;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import io.petstore.beans.Pet;
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.http.ContentType;
+
+@QuarkusTest
+public class PetStoreTest {
+
+    @Test
+    public void testapi() {
+
+        Pet pet = new Pet().withName("test").withId(1234L).withStatus(Pet.Status.available);
+
+        given()
```

Comment:
Hi @cmoine-swi, It is close, I think we can test the final state of `Pet` after building it. WDYT?

There is no need to make an HTTP request.